### PR TITLE
Modify repo for manuscript

### PR DESCRIPTION
**Pull request by kaitejohnson**
This PR modifies mainly the README.md file to orient a reader of the upcoming manuscript how to navigate the pipeline codebase.

This will include:
- removing the `model_definition.md` as it is out of date and this repo no longer contains model code
- describing each of the folders, including where the archives of the real-time submissions and the associated metadata are saved
- pointing to the commit history for the model used during the real-time submission

LMK thoughts on this and if the convoluted nature of our model + pipeline split into modeling repo (`wwinference`) and eval pipeline repo with no production repo is at all clear (probs not)

### CODE REVIEW

**Review by seabbs — DISMISSED** [review]
This is really good I think

File: README.md [hunk 1]
```diff
@@ -1,23 +1,49 @@
...
+|[`output/forecasts`](output/forecasts)| Records of the real-time forecasts and associated metadata submitted to the [COVID-19 Forecast Hub](https://github.com/reichlab/covid19-forecast-hub/tree/master) from February 5th through April 29th, 2024 |
+|[`output/benchmarking`](output/benchmarking)| Metrics of performance from different versions of the model, used to assess model changes |
+|[`src`](src)| Code used to generate the configuration files used in the Azure batch pipeline and the post-processing pipeline |
+|[`.github`](.github) | GitHub actions used to set up the CI for the `wweval` functions
+|[`docs`](docs)| Record of the evaluation plan, note this has not been updated to reflect the current state of the evaluation workflow |
+|[`scratch`](scratch)| Various scratch files used throughout the project |
+|[`_targets.R`](_targets.R)| Now deprecated real-time [targets](https://books.ropensci.org/targets/) pipeline |
+|[`model_diagnostics`](model_diagnostics)| R markdown presenting summaries of the model diagnostics for use in the ream-time production setting |
```
Comment (dylanhmorris) [inline README.md]:
```suggestion
|[`model_diagnostics`](model_diagnostics)| R markdown presenting summaries of the model diagnostics for use in the real-time production setting |
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,49 @@
...
-We are currently developing an R package called [wwinference](https://github.com/CDCgov/ww-inference-model) that uses the model developed here and wraps it in a user-friendly interface.
-We plan on using this package to produce regular forecasts and evaluate retrospective forecast performance.
-To try out running this model on your own data, we encourage you to check out that R package and codebase.
+# Bayesian generative modeling for heterogeneous wastewater data applied to COVID-19 forecasting
+This repository contains the code to generate the results of evaluating retrospectively the forecast performance of a wastewater-informed forecasting model, both compared to a model without wastewater data and compared to other models submitted to the  [COVID-19 Forecast Hub](https://github.com/reichlab/covid19-forecast-hub/tree/master) over the 2023-24 epidemic season.
+The model is run using the [`wwinference` R package](https://github.com/CDCgov/ww-inference-model), please see that GitHub repository for a [mathematical description](https://github.com/CDCgov/ww-inference-model/blob/main/model_definition.md) of the model and details on how to install the package and run the model.
```
Comment (dylanhmorris) [inline README.md]: Could link to the docs first and then the Github repo second if you wanted. Very optional.

```suggestion
The model is run using the [`wwinference` R package](https://cdcgov.github.io/ww-inference-model). Please see that package's [GitHub repository](https://github.com/cdcgov/ww-inference-model/) for a [mathematical description](https://github.com/CDCgov/ww-inference-model/blob/main/model_definition.md) of the model and details on how to install the package and run the model.
```

**Review by dylanhmorris — DISMISSED** [review]
This is great, thanks! Caught one typo. Also looks like pre-commit might need to be run locally.

In the future we could consider moving deprecated-but-paper-relevant stuff like `_targets.R` and the outdated evaluation plan to some kind of `archive` folder.

**Review by dylanhmorris — APPROVED** [review]
LGTM

**Pull request merged**